pom.xml

```diff
@@ -0,0 +1,389 @@
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+		<profile>
+			<id>bstack-local</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<argLine>-Dcucumber.options="src/test/resources/com/browserstack/${test-name}"
+							</argLine>
+							<systemPropertyVariables>
+								<environment>remote</environment>
+							</systemPropertyVariables>
+							<suiteXmlFiles>
+								<suiteXmlFile>src/test/resources/conf/TestNG_Single.xml</suiteXmlFile>
+							</suiteXmlFiles>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+		<profile>
+			<id>bstack-local-parallel</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<systemPropertyVariables>
+								<environment>remote</environment>
+							</systemPropertyVariables>
+							<suiteXmlFiles>
+								<suiteXmlFile>src/test/resources/conf/TestNG_Parallel.xml</suiteXmlFile>
+							</suiteXmlFiles>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+		<profile>
+			<id>bstack-local-parallel-browsers</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<systemPropertyVariables>
+								<environment>remote</environment>
+							</systemPropertyVariables>
+							<suiteXmlFiles>
+								<suiteXmlFile>src/test/resources/conf/TestNG_MultipleParallel.xml</suiteXmlFile>
+							</suiteXmlFiles>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+		<profile>
+			<id>bstack-parallel</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<systemPropertyVariables>
+								<environment>remote</environment>
+								<url>${application-url}</url>
+							</systemPropertyVariables>
+							<suiteXmlFiles>
+								<suiteXmlFile>src/test/resources/conf/TestNG_Parallel.xml</suiteXmlFile>
+							</suiteXmlFiles>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+		<profile>
+			<id>bstack-parallel-browsers</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<systemPropertyVariables>
+								<environment>remote</environment>
+								<url>${application-url}</url>
+							</systemPropertyVariables>
+							<suiteXmlFiles>
+								<suiteXmlFile>src/test/resources/conf/TestNG_MultipleParallel.xml</suiteXmlFile>
+							</suiteXmlFiles>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+	</profiles>
```

Comment:
environments are not being picked from browserstack.conf.json file, instead getting picked from the XML files and only running on chrome and firefox.